src/components/field-types/CheckboxFieldComponent.vue

```diff
@@ -17,6 +16,10 @@
           :disabled="field.disabled">
         <label :for="field.id + '-' + index" class="form-check-label">{{ option.label }}</label>
       </div>
+      <p>
```

Comment:
If you are using <p> for the line break, you can also create an extra `<div class="form-group">`